internal_testsuites/golang/testsuite/startosis_run_sh_task_test/run_task_sh_task_test.go

```diff
@@ -55,6 +55,11 @@ def run(plan):
   result2 = plan.run_sh(run="cat /temp/tech.txt | tr -d '\n'", files={"/temp": file_artifacts[0]})
   plan.verify(result2.output, "==", "kurtosis")
 `
+
+	runShWithEnvVar = `
+def run(plan):
+  result = plan.run_sh(run="mkdir -p kurtosis && echo $EXAMPLE",image="badouralix/curl-jq",env_vars={"EXAMPLE": "value"})
```

Comment:
maybe can add a plan.verify here to test that the value is value

Reply:
Awesome!